miso-web/src/main/webapp/pages/bulkEditLibraries.jsp

```diff
@@ -114,15 +114,12 @@
       };
 
       // get SampleOptions and make the appropriate table
-      // TODO: implement for plain sample
-      if (Boolean(Hot.detailedSample)) {
-        Hot.saveButton.addEventListener('click', Library.hot.saveData, true);
-        
-        if (Library.hot.propagateOrEdit == 'Propagate') {
-          Hot.fetchSampleOptions(Library.hot.makeBulkCreateTable);
-  	    } else {
-  	      Hot.fetchSampleOptions(Library.hot.makeBulkUpdateTable);
-	      }
+      Hot.saveButton.addEventListener('click', Library.hot.saveData, true);
+
+      if (Library.hot.propagateOrEdit == 'Propagate') {
+        Hot.fetchSampleOptions(Library.hot.makeBulkCreateTable);
```

Comment:
Err, yes and no. There are many data paths that still want to access that data. It's much easier to have them access a list of empty values than it is to protect all of them with guards that return an empty list.